apps/prairielearn/elements/pl-file-editor/pl-file-editor.py

```diff
@@ -167,7 +161,7 @@ def parse(element_html: str, data: pl.QuestionData) -> None:
     # Get submitted answer or return parse_error if it does not exist
     file_contents = data["submitted_answers"].get(answer_name, "")
     if not file_contents and not allow_blank:
-        add_format_error(data, "No submitted answer for {0}".format(file_name))
+        pl.add_files_format_error(data, "No submitted answer for {0}".format(file_name))
```

Comment:
```suggestion
        pl.add_files_format_error(data, f"No submitted answer for {file_name}")
```
Should switch to the more modern f-strings where possible.